pkg/acsengine/const.go

```diff
@@ -80,6 +80,28 @@ const (
 
 // KubeImages represents Docker images used for Kubernetes components based on Kubernetes version
 var KubeImages = map[api.OrchestratorVersion]map[string]string{
+	api.Kubernetes171: {
+		"hyperkube":       "hyperkube-amd64:v1.7.1",
+		"dashboard":       "kubernetes-dashboard-amd64:v1.6.1",
+		"exechealthz":     "exechealthz-amd64:1.2",
+		"addonresizer":    "addon-resizer:1.7",
+		"heapster":        "heapster:v1.3.0",
+		"dns":             "k8s-dns-kube-dns-amd64:1.14.4",
+		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
+		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.13.0",
+		"pause":           "pause-amd64:3.0",
+		"windowszip":      "v1.7.1intwinnat.zip",
```

Comment:
@JiangtianLi let's build up a Windows Kubernetes v1.7.1 zip. Windows support doesn't need to block this—we can roll out Linux support first.